### PR TITLE
[chore][mdatagen] Add flag to indicate that component is seeking new code owners

### PR DESCRIPTION
**Pull request by djaglowski**
See #31115

### CODE REVIEW

**Review by crobert-1 — APPROVED** [review]
Really like it, it definitely feels more welcoming to help 👍

File: cmd/mdatagen/templates/readme.md.tmpl [hunk 1]
```diff
@@ -23,7 +23,7 @@
 {{- if .Status.Codeowners }}
 {{- $codeowners := userLinks .Status.Codeowners.Active }}
 {{- $emeritus := userLinks .Status.Codeowners.Emeritus }}
-| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | {{ stringsJoin $codeowners ", " }} |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | {{ stringsJoin $codeowners ", " }} {{ if .Status.Codeowners.SeekingNew }}Seeking more code owners! {{ end }}|
```
[the remark this answers is not in the document]
Comment (djaglowski) [reply cmd/mdatagen/templates/readme.md.tmpl:26]: Updated.

**Comment by atoulme** [comment]
Would it be ok to add this to core as well? We're trying to delete mdatagen in contrib with #30495

**Comment by djaglowski** [comment]
> Would it be ok to add this to core as well? We're trying to delete mdatagen in contrib with #30495

Sure, I don't really care where it lives as long as we're generating the correct documentation. Can we have it here in the meantime until the switch is made?

**Review by songy23 — APPROVED** [review]

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**